fedora/Dockerfile.amd64

```diff
@@ -14,84 +57,49 @@ LABEL description="minimal Fedora with xfce4, VNC & squish to run GUI tests of t
     It's not intended for public use but purely for CI runs."
 
 ENV TZ=Europe/Berlin
-ENV TERM xterm
-ENV LANG C.UTF-8
-ENV LANGUAGE en_US:en
-ENV LC_ALL C.UTF-8
+ENV LANG=C.UTF-8
+ENV LANGUAGE=en_US:en
+ENV LC_ALL=C.UTF-8
 
 ENV FONTCONFIG_PATH=/etc/fonts
 ENV OWNBUILD_DIR=/ownbuild
 ENV CLIENT_BRANCH=${ARG_CLIENT_BRANCH:-master}
 ENV CLIENT_BUILD_TARGET=${ARG_CLIENT_BUILD_TARGET:-linux-gcc-x86_64-squish}
 
-RUN yum install -y \
-    wget \
-    git-core \
-    ffmpeg-free \
-    hostname \
-    # to build python3
-    openssl-devel \
-    cairo-devel \
+RUN dnf install -y --setopt=install_weak_deps=False \
+    # to build client
+    libEGL-devel \
+    gcc-c++ \
+    # for python3 modules
     gobject-introspection-devel \
     cairo-gobject-devel \
-    sqlite-devel \
     # used in GUI test code
     gdb \
     gnome-keyring \
     nautilus-python \
-    procps-ng \
     nodejs \
-    && npm install --silent -g pnpm \
+    pnpm \
     # cleanup
-    && yum autoremove -y \
-    && yum clean all
-
-### install Python 3.10 including needed modules
-RUN mkdir /python && \
-    cd /python && \
-    wget -O-  https://www.python.org/ftp/python/3.10.12/Python-3.10.12.tgz | tar xz --strip 1 -C . && \
-    ./configure --enable-optimizations --enable-shared --prefix=/usr/local LDFLAGS="-Wl,-rpath /usr/local/lib" && \
-    make && \
-    sudo make install && \
-    cp /python/libpython* /usr/lib && \
-    python3.10 -m pip install requests && \
-    python3.10 -m pip install pycairo && \
-    python3.10 -m pip install PyGObject && \
-    rm -rf /python
-
-FROM stage-fedora as stage-craft
-
-RUN yum install -y \
-    cmake \
-    ninja-build \
-    libEGL-devel \
-    gcc \
-    gcc-c++ \
-    libxkbcommon-devel \
+    && dnf autoremove -y \
+    && dnf clean all \
+    && dnf clean dbcache
+
+### install Python 3.10
+RUN curl https://dl.fedoraproject.org/pub/fedora/linux/releases/39/Everything/x86_64/os/Packages/p/python3.10-3.10.13-1.fc39.x86_64.rpm \
```

Comment:
wasn't python already installed in line 24?

Reply:
that is in the build stage. it wont be part of the final image